### PR TITLE
FW-142 Add skynet config that verifies that github actions passes

### PR DESCRIPTION
**Pull request by wesleybalvanz-wf**
This adds a skynet config that verifies that the github workflow passes so that the RM pipeline doesn't fail and no manual intervention is required.

### CODE REVIEW

File: skynet.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: verify-github-actions
+description: Verify that the github actions run passed, this is needed to make pipelines pass without manual intervention
+contact: 'Frontend Frameworks Architecture / #support-frontend-architecture'
+image: drydock.workiva.net/workiva/skynet-images:3693290 # Uses the image from this branch: https://github.com/Workiva/skynet-images/pull/127
```
Comment (wesleybalvanz-wf) [inline skynet.yaml:4]: If wanting to review the actual script it is located at the PR listed [here](https://github.com/Workiva/skynet-images/pull/127) ^

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by robbecker-wf** [comment]
QA+1 CI Passes and it verifies that the github run succeeds

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**